include/libtorrent/session_settings.hpp

```diff
@@ -1588,6 +1589,11 @@ namespace libtorrent
 		// the max number of packets per second a DHT node is allowed to send
 		// without getting banned.
 		int block_ratelimit;
+
+		// when set, this node no longer responds to 'query' messages that it 
+		// receives, and in each outgoing query message, place a 'ro' key in the 
+		// top-level message dictionary and sets its value to 1.
```

Comment:
would you mind also including a higher level description of the consequence of this is? like, you won't be kept in people's routing tables and it's meant for low-power and/or ephemeral devices that cannot support the DHT.